src/Poseidon/CLI/OptparseApplicativeParsers.hs

```diff
@@ -446,35 +446,37 @@ parseInGenoOne = OP.option (OP.eitherReader readGenoInput) (
         OP.short 'p' <>
         OP.metavar "FILE" <>
         OP.help "One of the input genotype data files. \
-                \Expects .bed, .bed.gz, .bim, .bim.gz or .fam for PLINK, or \
-                \.geno, .geno.gz, .snp, .snp.gz or .ind for EIGENSTRAT. \
-                \The other files must be in the same directory \
+                \Expects .bed, .bed.gz, .bim, .bim.gz or .fam for PLINK, \
+                \.geno, .geno.gz, .snp, .snp.gz or .ind for EIGENSTRAT, or\
+                \.vcf or .vcf.gz for VCF. \
+                \In case of EIGENSTRAT and PLINK, the two other files must be in the same directory \
                 \and must have the same base name. If a gzipped file is given, it is assumed that the \
                 \file pairs (.geno.gz, .snp.gz) or (.bim.gz, .bed.gz) are both zipped, but not the .fam or .ind file. \
-                \If a .ind or .fam file is given, it is assumed that none of the file triples is zipped. \
-                \For VCF please see option --vcfFile")
+                \If a .ind or .fam file is given, it is assumed that none of the file triples is zipped.")
     where
         readGenoInput :: FilePath -> Either String GenotypeFileSpec
         readGenoInput p =
             let (path, extension) = splitExtensionsOptGz p
             in  makeGenoInput path extension
         makeGenoInput path ext
             | ext `elem` [".geno",    ".snp",   ".ind"] =
-                Right $ GenotypeEigenstrat (path <.> ".geno")    Nothing
-                                           (path <.> ".snp")     Nothing
-                                           (path <.> ".ind")     Nothing
+                Right $ GenotypeEigenstrat (path <> ".geno")    Nothing
```

Comment:
But... why? `<.>` is neat...

Reply:
True. Don't know why I opted for that. I've brought it back now.